.github/workflows/test.yml

```
@@ -11,16 +11,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["2.7", "3.5", "3.6", "3.7", "3.8", "3.9", "3.10-dev", "pypy3"]
+        python-version: ["2.7", "3.5", "3.6", "3.7", "3.8", "3.9", "3.10", "3.11", "pypy3"]
```

Maybe test `3.12-dev`?
```suggestion
        python-version: ["2.7", "3.5", "3.6", "3.7", "3.8", "3.9", "3.10", "3.11", "3.12-dev", "pypy3"]
```

---

Done in https://github.com/xolox/python-humanfriendly/pull/69/commits/11d62fbb1548cd4280cb4bdecae5d70910f8910d